zerver/webhooks/zapier/doc.html

```diff
@@ -40,3 +40,36 @@
 <p>You're done! Example message:</p>
 
 <img class="screenshot" src="/static/images/integrations/zapier/005.png"/>
+
+<p>
+    <b>Weather Update with Zapier!</b><br />
+</p>
+<p>
+    Go to the following link <code>https://zapier.com/app/explore</code> after you sign into your Zapier account<br />
+   Select "Send weather info to your app via webhooks".
+
+    <img class="screenshot" src="/static/images/integrations/zapier/006.png" /><br />
+    Choose the trigger app <b>Weather by Zapier</b><br /><br />
+    <img class="screenshot" src="/static/images/integrations/zapier/007.png" /><br />
+    Next, select the option <b>Today's Forecast</b><br /><br />
+    <img class="screenshot" src="/static/images/integrations/zapier/008.png" /><br />
+    Fill in the latitude and longitude of your location.<br /><br />
+    <img class="screenshot" src="/static/images/integrations/zapier/009.png" /><br />
+    After you select <code>Webhook</code> as the action, choose the  <code>POST</code> option
```

Comment:
Should you remind them to specify their units (Celsius versus Fahrenheit)? Also, will the output of the integration include an F or a C when it mentions temperatures?